exporter/exporterhelper/internal/batch_sender_test.go

```diff
@@ -158,7 +158,6 @@ func TestBatchSender_BatchExportError(t *testing.T) {
 			assert.Eventually(t, func() bool {
 				return sink.requestsCount.Load() == tt.expectedRequests &&
 					sink.itemsCount.Load() == tt.expectedItems &&
-					be.BatchSender.(*BatchSender).activeRequests.Load() == 0 &&
```

Comment:
Why is this line removed? We should wait until there are no in-flight requests

Reply:
This is because if the new feature is turned on, we use `queueBatcher` instead of `BatchSender` so `BatchSender` is a nil. In the previous PR, I set up the test to run with both the feature on and off so this line has to be removed.

Reply:
Ok sounds good 👍